src/remediations/controller.read.js

```diff
@@ -204,7 +204,7 @@ exports.list = errors.async(async function (req, res) {
                 req.user.username
             );
 
-            playbook_runs = playbook_runs?.toJSON();
+            playbook_runs = playbook_runs?.toJSON() || {};
```

Comment:
If there are no playbook runs, don't we want to return right away?

Reply:
Well... sort of.  This code is really bad about side effects so you have to be really careful, but we can at least break out of this 'if' block.  I was in the process of changing that when I got pulled into a series of meetings but I'll push an update here shortly...